src/core/rtkQuery/splitV3Api/seriesApi.ts

```diff
@@ -81,6 +85,16 @@ const seriesApi = splitV3Api.injectEndpoints({
       providesTags: ['SeriesEpisodes', 'UtilitiesRefresh'],
     }),
 
+    // Toggles the watched state for all the episodes that fit the query.
```

Comment:
Does it toggle or does it set watched to specified value? Because toggle means to switch the status to next one usually, so phrasing is confusing.

Reply:
I copied the comment from the server 🤷 .
We can change it on our side

Reply:
It toggles it on or off. Or so my late night brain was probably thinking when it made my fingers write thst comment. 🧠